### PR TITLE
remove objc code, add right example

### PR DESCRIPTION
**Pull request by Sarkar**
@qinweigong @derrickstaten

### CODE REVIEW

File: android-app-content-share-with-deeplink.md [hunk 1]
```diff
@@ -220,45 +220,47 @@ You should now help the user share this on Facebook, via SMS, or through whichev
...
-        // Load the next logical view controller
-        } else {
-            nextVC = [storyboard instantiateViewControllerWithIdentifier:@"MainViewController"];
+The following implementation can tell if a user wanted to view a picture of an existing monster - even if the user just installed the app and this is the first open! If the user clicked on a Branch link with the parameter _monster_ attached, the application redirects to a screen to view the picture. In addition, this user can be shown a personalized message, such as the following: "Thanks for checking out our app. Let's view the picture that Mario just shared with you." Otherwise the default view controller is shown. Obviously routing logic is heavily implementation-specific, so the code below is just an example. See our Android sample project [Branchster](https://github.com/BranchMetrics/Branchster-Android) for the full example on routing.
+
+```java
+
+protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    branch = Branch.getInstance(this.getApplicationContext(), getResources().getString(R.string.bnc_app_key));
```
Comment (qinweigong) [inline android-app-content-share-with-deeplink.md]: The second param is not necessary, just branch = Branch.getInstance(this.getApplicationContext());

File: android-app-content-share-with-deeplink.md [hunk 2]
```diff
@@ -220,45 +220,47 @@ You should now help the user share this on Facebook, via SMS, or through whichev
...
+    branch.initSession(new BranchReferralInitListener() {
+        @Override
+        public void onInitFinished(JSONObject referringParams, BranchError error) {
+            Log.i("Branchster", "branch init complete!");
```
Comment (qinweigong) [inline android-app-content-share-with-deeplink.md]: You might want to check error first, to give a good example

**Comment by aaustin** [comment]
:+1:

File: android-app-content-share-with-deeplink.md (hunk 2 above)
Comment (Sarkar) [inline android-app-content-share-with-deeplink.md]: good point, adding this now and defaulting it to start a general activity.

**Pull request merged**